### PR TITLE
Pension 69961 mvp current spouse monthly support

### PR DESCRIPTION
**Pull request by ToddWebDev**
## Summary
On the Pension Benefits application(21P-527EZ), update the 'Health and employment information' chapter

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Go to the 'Household Information' chapter
4. Select 'Married' or 'Separated' on the 'Marital status' page
5. Continue and complete the 'Current spouse's name' page
7. Continue and complete the 'Current marriage information' page
8. Continue and verify the 'Financial support for your spouse' page is next
9. Verify the content and design of the page
9. Confirm that the input is required
10. Verify that all changes are saved to the redux store in form data and included in the save form api call

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69961)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done
- [x] Existing unit tests updated
- [x] New unit tests added

### Desktop
| Financial support for your spouse | Financial support for your spouse (Required) |
| ------ | ------ |
|localhost_3001_pension_application_527EZ_household_marital-status_spouse-monthly-support|localhost_3001_pension_application_527EZ_household_marital-status_spouse-monthly-support (1)

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69961)

### Quality Assurance & Testing
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/pensions/config/form.js [hunk 1]
```diff
@@ -440,6 +441,13 @@ const formConfig = {
           uiSchema: dateOfCurrentMarriage.uiSchema,
           schema: dateOfCurrentMarriage.schema,
         },
+        currentSpouseMonthlySupport: {
+          title: 'Financial support for you spouse',
```
Comment (TaiWilkin) [inline src/applications/pensions/config/form.js:445]: you -> your

**Review by TaiWilkin — APPROVED** [review]
LGTM
One small spelling error that doesn't show up in the UI

**Review by TaiWilkin — DISMISSED** [review]

**Review by TaiWilkin — APPROVED** [review]
LGTM

File: src/applications/pensions/config/form.js (hunk 1 above)
Comment (zack) [reply src/applications/pensions/config/form.js:457]: @ToddWebDev looks like you missed this comment and it got merged with a typo :)

Comment (ToddWebDev) [reply src/applications/pensions/config/form.js:457]: @zack This spelling error won't show up in the UI. Will fix in a later commit though.

**Pull request merged**